CHANGELOG.md

```diff
@@ -1,8 +1,16 @@
+<<<<<<< HEAD
```

Comment:
YOu have a merge line here. Please remove it

Reply:
OMG, sure, sorry :(